src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Leaderboard</title>
+</head>
+<body>
+  <h1>Leaderboard</h1>
+  <div class="content">
+    <div class="left-content">
+      <div class="table-header">
+        <h3 class="scores-header">Recent scores</h3>
+        <button class="refresh-btn" id="refresh" type="button">Refresh</button>
+      </div>
+      <table class="scores">
+        
```

Comment:
- Kindly provide items of your chart: (Name, Score).
![leaderboard_wireframe](https://user-images.githubusercontent.com/22258274/131780076-bcd70792-2931-4c52-b9a9-3928ee05a781.png)

Reply:
This issue is well addressed! :dancers: